doc/api/deprecations.md

```diff
@@ -3782,9 +3782,12 @@ changes:
   - version: v23.4.0
     pr-url: https://github.com/nodejs/node/pull/55892
     description: Documentation-only.
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/55753
+    description: Runtime deprecation.
```

Comment:
```suggestion
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/55753
    description: Runtime deprecation.
  - version: v23.4.0
    pr-url: https://github.com/nodejs/node/pull/55892
    description: Documentation-only.
```
The order of entries here should be reversed, most recent change must be listed first.